lib/actions/install.js

```diff
@@ -101,12 +101,14 @@ install.installDependencies = function (options) {
     };
   }
 
-  options = _.defaults(options || {}, {
-    bower: true,
-    npm: true,
-    skipMessage: false,
-    callback: function () {}
-  });
+  if (!options) {
```

Comment:
Maybe the right approach then is to not set default values for these and check if they're specifically set as false or true.